bin/ch-fromhost

```diff
@@ -337,7 +337,6 @@ if [ -n "$cray_fi_found" ]; then
             continue
         fi
         enqueue_file "$l"
-        [ ! -L "$l" ] || enqueue_file "$(readlink -f "$f")"
```

Comment:
Why was this line removed? Looks like some kind of symlink handling?

Reply:
> Does it rebuild the two symlinks in the container though?

It does not. FWIW, it does not appear to matter. See below.

```
$ ./bin/ch-fromhost -v -p /qux/libfabric.so /tmp/ompi/
[ debug ] enqueue file(s)
[ debug ]    shared library: /foo/bar/libdbz.so.9000
[...]
[ debug ] injecting into image: /tmp/ompi/
[ debug ]    /foo/bar/libdbz.9000 -> /usr/local/lib (inferred)
done
$ ./bin/ch-run /tmp/ompi/ -- ldd /usr/local/lib/libfabric.so
[...]
	libdbz.so.9000 => /usr/local/lib/libdbz.so.9000 (0x10101robotsolo-thehumansaredead)
[...]
```

The overwritten library in the container, `/usr/local/lib/libfabric.so`, still resolves the library dependency, and runs as expected on single and multiple nodes.

Reply:
The same is true for the other libraries pulled in.

Reply:
Looks like `ldconfig` will make the appropriate symlinks, and we run that anyway.